ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/services/ApiSubmissionService.scala

```diff
@@ -131,20 +131,20 @@ final class ApiSubmissionService private (
                 Success(())
 
               case Success(Overloaded) =>
-                logger.debug(s"Submission has failed due to back pressure")
+                logger.info(s"Submission has failed due to back pressure")
```

Comment:
Unnecessary string interpolation.

Reply:
Took care of this, and others around it. Thanks!